openedx/features/redhouse_panel/templates/redhouse_panel/edx_ace/passwordset/email/body.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% trans "Invitation to Join " %} {{ platform_name }}
+            </h1>
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}An account has been created for you in {{ platform_name }}. Please use the link below to set your password and log in.{% endblocktrans %}
+                <br />
+            </p>
+
+            {% if failed %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% trans "If you don't want to join, you can ignore this email." %}
+                    <br />
+                </p>
```

Comment:
Could you please elaborate the purpose of this check? in what case this `failed` would be true?

Reply:
@danialmalik  this was added in the "password reset" email  for when user doesn't exist, but I guess this could be removed in Password set email as we are not passing `failed` from anywhere